### DISCUSSION TITLE
Student benefits

### DISCUSSION THREAD

**Discussion by vanesagab** [community/community#30078] [GitHub Education] [answered]
What are the student benefits and how to get them?

**Comment by manasesjesus** [comment, marked as answer]
There are many benefits in the GitHub Student Developer Pack. You can check all of them here: https://education.github.com/pack

To sign up for the benefits, first go to Settings > Emails and provide your university email. Personal or work emails aren't allowed; it must be from an educational institution. Make sure it's added and verified in your emails: https://github.com/settings/emails

You need to activate location services on your browser and go to this webpage: https://education.github.com/discount_requests/student_application
Providing your current location helps GitHub to verify your affiliation with your chosen school/university. You are far less likely to be verified if you choose not to share your location.

Provide the other requested data, and that's it. You'll get the GitHub student benefits for two years. After that, if you're still a student, you can renew it.

I had it four years while being a student.
<img width="321" alt="gsp" src="https://user-images.githubusercontent.com/24204142/185619291-261315c6-5b4d-446f-88e9-94042cc3a05f.png">

Good luck!

**Reply by vanesagab** [reply]
I will try it. Thanks!

**Reply by abdulrahman-hegazy** [reply]
Hi @manasesjesus can you help me over here
https://github.com/orgs/community/discussions/28390
It would be a life changing opportunity

**Reply by AdiAr11** [reply]
Hi, I want to apply for students benefits program. I did apply for it in 2020, but that one will end in few days. I am trying to apply again but it says I need college issue id, but I don't have one. Our college doesn't provide it. I can provide other documents like my ID card.
